app/views/login.ejs

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+  <link href="stylesheet/login.css" rel="stylesheet" type="text/css">
+    <head>
+        <title>
+        </title>
+        <style type="text/css">
```

Comment:
put this into the stylesheet, we want to avoid in page and in-line styles as much as we can.